jdatetime/__init__.py

```diff
@@ -942,6 +943,29 @@ def strptime(date_string, format):
                     "time data '%s' does not match format '%s'" %
                     (date_string, format)
                 )
+        z = get('z', None)
+        if z:
+            if z[3] == ':':
```

Comment:
Could you move this to another function? it improve the code readability

Reply:
its no problem.
Just to be clear this new function should be a static method of other classes (datetime, date, etc) or just a normal function outside of other classes.

Reply:
Since this is only used in datetime, just a "private" method on this class, as we don't want to expose to user of this lib

Reply:
I changed to a new static method in datetime class but used underscore prefix.
it couldn't be a normal private method of datetime class because then we can't call it from strpitime (which is a static method) without first initializing datetime class.
hope this is okay but I'm open to other suggestions to make it better.

Reply:
Looks good to me, as it's `_` function no one supposed to rely on that and we will change it later if necessary.

Last request could change the if condition in this function to : 
```python
        z = timezone_string  # keep the original string for value error exception.
        if z==None: return None
        ....
```
removes extra indention and simplify the code as well.

Reply:
I changed to:
```
def _timezone_from_string(timezone_string):
    if timezone_string is None:
        return None
```
`is` is a better solution because its faster as it's a binary comparison.
and I tried to make it a one line but flake8 check wouldn't pass so I made it 2 lines with indentation.